lib/pos_epoch_complete_hook_test.go

```diff
@@ -303,8 +303,8 @@ func TestRunEpochCompleteHook(t *testing.T) {
 		require.Len(t, stakeEntries, 7)
```

Comment:
Missing a diff that I see in #1163:
```
require.Equal(t, snapshotValidatorSetTotalStakeAmountNanos, uint256.NewInt().SetUint64(2800))
```

This is OK since we deleted the SnapshotValidatorSetToTotalStakeAmountNanos db prefix.